ingestion_server/test/run_test.sh

```diff
@@ -18,8 +18,6 @@ succeeded=$?
 if [[ $succeeded -eq 0 ]]; then
 	printf "${green}:-) All tests passed${endcol}\n"
 else
-  printf "Full system logs:\n"
-  cat test/ingestion_logs.txt
 	printf "${red}:'( Some tests did not pass${endcol}\n"
```

Comment:
Just for consistency between the two `run_test.sh` scripts, we could add the same message here with the hint to use `just logs`.